Makefile

```diff
@@ -92,3 +92,8 @@ doctest:
 	$(SPHINXBUILD) -b doctest $(ALLSPHINXOPTS) $(BUILDDIR)/doctest
 	@echo "Testing of doctests in the sources finished, look at the " \
 	      "results in $(BUILDDIR)/doctest/output.txt."
+
+pdf:
+	$(SPHINXBUILD) -b pdf $(ALLSPHINXOPTS) _build/pdf
+	@echo
+	@echo "Build finished. The PDF Version of Documentation is in _build/pdf."
```

Comment:
There's no newline character on the last line of this file in your new version (as you can see in the github rendering of the patch - hover your mouse cursor over the slashed-zero).

It's best to have text files end with a newline character - some tools don't handle such files nicely.  And in general, invisible differences in files aren't helpful - trailing whitespace on lines is another example.